test/hotspot/jtreg/vmTestbase/nsk/jdi/ObjectReference/setValue/setvalue004t.java

```diff
@@ -42,6 +42,17 @@ public class setvalue004t {
     static final boolean sBooleanFld = false;
     static final String  sStrFld = "instance field";
 
+    // tested static final fields
+    static final byte    iByteFld = 127;
+    static final short   iShortFld = -32768;
+    static final int     iIntFld = 2147483647;
+    static final long    iLongFld = 9223372036854775807L;
+    static final float   iFloatFld = 5.1F;
+    static final double  iDoubleFld = 6.2D;
+    static final char    iCharFld = 'a';
+    static final boolean iBooleanFld = false;
+    static final String  iStrFld = "instance field";
```

Comment:
I don't think you meant for these fields to be static.

Reply:
Oh, sure. Incomplete change (or I did undo accidentally)

Reply:
Fixed.